src/main/java/com/acoustic/SpringPolandSalaryCalculator/calculatorservice/AnnualGrossService.java

```diff
@@ -0,0 +1,33 @@
+package com.acoustic.SpringPolandSalaryCalculator.calculatorservice;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+import org.springframework.stereotype.Component;
+
+import lombok.RequiredArgsConstructor;
+
+
+@Component
+@RequiredArgsConstructor
+public class AnnualGrossService implements SalaryCalculatorService {
```

Comment:
What do you think to change the annotation to @Service? To be consistent with class name :)

Reply:
I have replaced the @Component with @Service annotation, if i remove the annotations @Service the salary-calculations endpoint will just return the average without the other calculations